Plugins/NativeCamera/Source/Apple/CameraDevice.mm

```diff
@@ -20,10 +20,15 @@
 
 @class CameraTextureDelegate;
 
-@interface CameraTextureDelegate : NSObject <AVCaptureVideoDataOutputSampleBufferDelegate>
+@interface CameraTextureDelegate : NSObject <AVCaptureVideoDataOutputSampleBufferDelegate, NSObject>
 {
+#if (__MAC_OS_X_VERSION_MIN_REQUIRED >= 140000 || __IPHONE_OS_VERSION_MIN_REQUIRED >= 170000)
+    // angle in half PI (0 = 0, 1 = 0.5PI, 2 = PI, 3 = 1.5PI)
+    // Might change in the future for a float.
+    @public int videoRotationAngle;
```

Comment:
Should be UpperCamelCase since it's public? I also don't understand the purpose of this int. If Apple is deprecating the enum, can we introduce our own? Why int?

Reply:
Replaced with an enum